types.ts

```diff
@@ -1172,6 +1172,134 @@ export interface RateLimits {
  */
 export type BasicPackDefinition = Omit<PackVersionDefinition, 'version'>;
 
+/**
+ *  Default tool types supported by Coda Packs for job definitions.
+ * @hidden
+ */
+export enum ToolTypes {
+  /**
+   * Allows formulas within a pack to be used as tools.
+   */
+  Pack = 'Pack',
+  /**
+   * Allows knowledge to be used as a tool.
+   */
+  Knowledge = 'Knowledge',
+}
+
+/**
+ * The type identifier for a tool
+ * @hidden
+ */
+export type ToolType = keyof ToolMap;
```

Comment:
Would rename this to avoid the name conflict, maybe `AllToolTypes` or something.